backend/btrixcloud/invites.py

```diff
@@ -274,14 +274,17 @@ async def get_invite_out(
             return invite_out
 
         org = await users.org_ops.get_org_for_user_by_id(invite.oid, inviter)
+        if not org:
+            raise HTTPException(status_code=400, detail="invalid_invite")
+
         invite_out.orgName = org.name
         invite_out.orgSlug = org.slug
 
         if include_first_org_admin:
-            invite_out.firstOrgOwner = True
+            invite_out.firstOrgAdmin = True
```

Comment:
Does this require a frontend change?

Reply:
no, actually frontend is checking for `firstOrgAdmin` already, but this was changed to `firstOrgOwner`, so changing it back now to match frontend.